pkgs/by-name/br/brainflow/package.nix

```diff
@@ -0,0 +1,101 @@
+{
+  lib,
+  stdenv,
+  stdenvNoCC,
+  fetchFromGitHub,
+  fetchpatch,
+  bluez,
+  cmake,
+  dbus,
+  libftdi1,
+  nix-update-script,
+  pkg-config,
+  useLibFTDI ? true,
+  useOpenMP ? true,
+  buildBluetooth ? true,
+  buildBluetoothLowEnergy ? true,
+  buildONNX ? true,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "brainflow";
+  version = "5.12.1";
+
+  # This is done so that language-specific versions of the library can use sourceRoot, avoiding an IFD.
```

Comment:
this is not necessary as you are using `brainflow.src.name` in python module 

you probably did it because patches didnt apply when using sourceRoot

you can use something like 

```
  preBuild = ''
    cd python_package
  '';

I've seen several people discourage usage of sourceRoot
```